docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/server-default-log-level.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Strapi Server log level configuration
+description: Strapi 5 allows users greater control over the level of detail in of their server logs
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - configuration
+ - middlewares
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Strapi Server log level configuration
+
+You can adjust the server log level in the configuration to control how much detail you see in your server logs. If you want to see more or less verbose logs in your server logs, this feature allows you to customize it according to your needs.
+
+<Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+The log level defaults to `silly`, which means that every log is shown, providing the most detailed information.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi 5**
+
+The log level defaults to `info`. This means that `silly`and `debug` level logs are hidden by default, offering a less verbose log output.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+- Open your `server.js` file.
```

Comment:
These actually are manual migration steps 😄 
So I'll turn lines 48-50 into an ordered list.
Line 51 is a real note, though.

***

Suggestion:

### Notes

* The log level can be configured either in the `server.js` file as described in the following Manual migration section, or as described in [configurations](/dev-docs/configurations/middlewares#logger).

### Manual migration

To update to Strapi 5, (please add steps here)

***

By the way, I'd replace "Open your `server.js` file" by "Open your server configuration file (`config/server`)" because: a) this way people know where to find it, and b) it can actually be .js or .ts file depending on whether they use TypeScript or not.